blueman/main/Sendto.py

```diff
@@ -232,7 +232,8 @@ def on_transfer_error(self, _transfer, msg=""):
             d.add_button(_("Retry"), Gtk.ResponseType.YES)
             d.add_button("_Cancel", Gtk.ResponseType.CANCEL)
 
-            self.client.remove_session(self.object_push.get_object_path())
+            if self.object_push:
+                self.client.remove_session(self.object_push.get_object_path())
```

Comment:
``self.object_push`` is created when a session is added. And the transfer can only error after the session is created. So why is there a need to check?

Reply:
Like before, the check won't hurt but I still wonder why mypy can't figure this out.

Reply:
You cannot derive the expected state from our code as this is triggered by a D-Bus event which could theoretically happen at any time, even if `self.object_push` is `None`, i.e. before a `session-added` signal got processed or after a `session-removed` signal.